addons/addon-base-raas/packages/base-raas-services/lib/schema/update-aws-accounts.json

```diff
@@ -49,6 +49,21 @@
     "encryptionKeyArn": {
       "type": "string",
       "pattern": "^arn:aws:kms:.*$"
+    },
+    "needsPermissionUpdate": {
+      "type": "boolean"
+    },
+    "createdAt": {
```

Comment:
These four fields are auto generated by DDB, it should not be passed in from the call. (e.g. the createdBy field could be used for audit, and we don't want ppl to change that value as they wish) For update API method, only the fields to be updated and the `required` fields in the schema need to be passed in.

Reply:
gotcha, thank you! I'll remove those.